Gemfile

```diff
@@ -28,6 +28,8 @@ gem 'jbuilder', '~> 2.0'
 gem 'cancancan', '~> 1.10'
 # bundle exec rake doc:rails generates the API under doc/api.
 gem 'sdoc', '~> 0.4.0', group: :doc
+# Use kaminari for pagination
+gem 'kaminari'
```

Comment:
Since you didn't do anything with Kaminari in this PR do you want to add a new ticket about adding kaminari pagination?